test/model/content_test.dart

```diff
@@ -306,13 +326,13 @@ class ContentExample {
         '<p><em>italic</em> <a href="https://zulip.com/">zulip</a></p>\n'
         '</div></div>',
     [SpoilerNode(
-      header: [ListNode(ListStyle.ordered, [
-        [ListNode(ListStyle.unordered, [
-          [HeadingNode(level: HeadingLevel.h2, links: null, nodes: [
-            TextNode('hello'),
+      header: [OrderedListNode([[
+          UnorderedListNode([
+            [HeadingNode(level: HeadingLevel.h2, links: null, nodes: [
+              TextNode('hello'),
+            ])]
           ])]
-        ])],
-      ])],
+        ], start: 1)],
```

Comment:
nit: keep `start: 1` as the first argument, before the positional `items` argument

Dart has the handy feature of allowing named and positional arguments to be interleaved in any order, including named first. That's useful to do in a case like this where there's a named argument that's short and fits easily on the first line and then a longer multi-line argument that's positional.